src/Currency.php

```diff
@@ -40,6 +40,9 @@ class Currency implements Arrayable, JsonSerializable
     /** How many decimals of the currency's values should get rounded. */
     protected int $rounding;
 
+    /** Setting to decide if trailing decimal zeros should be removed for when calling formatted(). */
+    protected bool $deleteTrailingDecimalZeros;
```

Comment:
- `trimTrailingDecimalZeros` is probably better
- The comment grammar should be fixed